smart-contracts/contracts/cl-vault/src/test_tube/proptest.rs

```diff
@@ -408,43 +457,20 @@ mod tests {
         numeric_part.parse::<u128>().unwrap()
     }
 
-    // ASSERT METHODS
-
-    // TODO: REMOVE THIS DEPRECATED
-    /*fn assert_deposit_withdraw(
-        wasm: &Wasm<OsmosisTestApp>,
-        contract_address: &Addr,
-        accounts: &Vec<SigningAccount>,
-        accounts_shares_balance: HashMap<String, Uint128>,
-    ) {
-        // TODO: multi-query foreach user created previously
-        for account in accounts {
-            let shares = get_user_shares_balance(wasm, contract_address, account);
-
-            // Check that the current account iterated shares balance is the same we expect from Hashmap
-            assert_eq!(
-                shares.balance,
-                accounts_shares_balance.get(&account.address()).unwrap()
-            );
-        }
-    }*/
-
-    /*
-    fn assert_swap() {
-        todo!()
-    }
+    // COMPOSE STRATEGY
 
-    fn assert_update_range() {
-        todo!()
+    // Actions enum
+    #[derive(Clone, Copy, Debug)]
+    enum Action {
+        Deposit,
+        Withdraw,
+        // Swap,
+        UpdateRange,
     }
-    */
-
-    // COMPOSE STRATEGY
 
     // get_initial_range generates random lower and upper ticks for the initial position
     prop_compose! {
-        // TODO: evaluate if lower_tick and upper_tick are too much arbitrary
-        fn get_initial_range()(lower_tick in 1i64..1_000_000, upper_tick in 1_000_001i64..2_000_000) -> (i64, i64) {
+        fn get_initial_range()(lower_tick in -1_000_000i64..1_000_000, upper_tick in 1_000_001i64..2_000_000) -> (i64, i64) {
```

Comment:
These are still abritrary ticks right? I think we want to use MIN_TICK and MAX_TICK from math/tick.rs here